streaming-connect-common/src/main/java/com/adobe/platform/streaming/auth/AbstractAuthProvider.java

```diff
@@ -59,7 +59,7 @@ public TokenResponse getContent(HttpConnection conn) throws HttpException {
         try (InputStream in = conn.getInputStream()) {
           return mapper.readValue(in, TokenResponse.class);
         } catch (HttpException | IOException e) {
-          throw new HttpException("Error parsing response", e);
+          throw new HttpException("Error parsing response", 405, e);
```

Comment:
In case parsing exception we can just throw 500 in place of 405 as with failed token we won't be able to send the payload, ideally we should stop connector in this case.

Reply:
This would mean that any malformed message (back from Token/Auth service) will cause the sink connector to stop. 
In hindsight that seems to be the right thing to do - will change